Backend/purrfect-match-backend/src/main/java/com/restservice/ContactPageModel/ContactPage.java

```diff
@@ -0,0 +1,85 @@
+package com.restservice.ContactPageModel;
+
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class ContactPage {
+    private String firstName;
+    private String lastName;
+    private String subject;
+    private String email;
+    private String shelterEmployee;
+    private String message;
+
+    ContactPage() {
+    }
+
+    public ContactPage(String firstName, String lastName, String subject, String email, String shelterEmployee,
```

Comment:
Was there talk about this using an enum for things that use a dropdown box/the shelterEmployee?